finder-app/finder.sh

```diff
@@ -0,0 +1,43 @@
+#ECEN-5713 Advanced Embedded Software Development
+#Author name: Khyati Satta
+#Date: 01/22/2023
+#File Description: Finder script to find number of files in a given directory containing a given expression
+
+#!/bin/bash
+
+#Store the first(the directory) and second(the string to search in the files)command line arguments into variables 
+direc=$1
+exp=$2
+
+#Variables for storing the output: Number of files in the directory and the number of matching lines in the files
+n_files=0
+n_lines=0
+
+#Error check 1: Check if number of arguments is 2
+if [ $# -ne 2 ]
+then
+    echo Please enter both arguments
```

Comment:
@KhyatiSatta how could you make this error message more descriptive?

Reply:
@dwalkes  I can definitely specify what are the expected arguments!